site/app/templates/AutogradingStatus.twig

```diff
@@ -0,0 +1,192 @@
+    </p>
+    <button class="btn btn-primary" id="toggle-btn">
+        Pause Update
+    </button>
+</div>
+<div class="content">
+    <h1>
+        Job Statistics
+    </h1>
+    <h2>
+        Course Statistics
+    </h2>
+    <table class="table table-striped mobile-table" id="course-table">
+        <thead>
+            <tr>
+                <th>
+                    Semester
+                </th>
+                <th>
+                    Course
+                </th>
+                <th>
+                    Gradeable Id
+                </th>
+                <th>
+                    Interactive
+                </th>
+                <th>
+                    Regrade
+                </th>
+            </tr>
+        </thead>
+        <tbody>
+        {% for sem_course, info in progress["course_info"] %}
+            {% for gradeable, types in info %}
+                <tr>
+                    {% set course_name = sem_course|split("__") %}
+                    <td>{{ course_name[0] }}</td>
+                    <td>{{ course_name[1] }}</td>
+                    <td>{{ gradeable }}</td>
+                    <td>{{ types["interactive"] }}</td>
+                    <td>{{ types["regrade"] }}</td>
+                </tr>
+            {% endfor %}
+        {% endfor %}
+        </tbody>
+    </table>
+    <h2 style="padding-top: 1em;">
+        Machine Statistics
+    </h2>
+    <table class="table table-striped" id="machine-table">
+        <thead>
+            <tr>
+                <th>Machine</th>
+                <th>Semester</th>
+                <th>Course</th>
+                <th>Gradeable Id</th>
+                <th>User Id</th>
+                <th>Elapsed Time</th>
+                <th>Error</th>
+            </tr>
+        </thead>
+        <tbody>
+            {% for machine, worker_info in progress["ongoing_job_info"] %}
+                    {% for info in worker_info %}
+                    <tr>
+                        <td> {{machine}} </td>
+                        <td> {{info["semester"]}} </td>
+                        <td> {{info["course"]}} </td>
+                        <td> {{info["gradeable_id"]}} </td>
+                        <td> {{info["user_id"]}} </td>
+                        <td> {{info["elapsed_time"]}} </td>
+                        <td> {{info["error"]}} </td>
+                    </tr>
+                    {% endfor %}
+            {% endfor %}
+        </tbody>
+    </table>
+</div>
+
+<div class="content table-content custom-scrollbar">
+    <h1>
+        Autograding Monitor
+    </h1>
+    <div id="data" data-machine-num="{{progress["machine_grading_counts"]|keys|length}}" data-capability-num="{{progress["capability_queue_counts"]|keys|length}}" hidden>
+    </div>
+    <table class="table table-striped table-bordered" id="autograding-status-table">
+        <thead>
+            <tr>
+                <th rowspan="3" class="col-md-2 right-boarder">
```

Comment:
Do we need a custom border class here (and elsewhere in this table) if we have the `.table-bordered` bootstrap class?

Reply:
The border is for the borders between the columns, which does not exist between every column. For example, between the machines and capabilities, each capability is a column but for the sake of association, I did not put a border between them.
![image](https://user-images.githubusercontent.com/49821332/129624721-4e923472-04b9-403d-a3d9-a9c3112c182a.png)